tests/settings.py

```diff
@@ -52,14 +53,15 @@ def _load_skaffold(path):
         skaffold = yaml.load(f, Loader=yaml.Loader)
 
     services = skaffold['deploy']['helm']['releases']
-    backends = [s for s in services if s['name'].startswith('substrabac-peer')]
+    backends = [s for s in services if s['name'].startswith('substra-backend-peer')]
 
     nodes = [NodeCfg(
         name=b['name'],
         msp_id=b['overrides']['peer']['mspID'],
         address=b['overrides']['backend']['defaultDomain'],
         user=b['overrides']['backend']['auth']['user'],
         password=b['overrides']['backend']['auth']['password'],
+        shared_path=b['overrides']['persistence']['hostPath'],
```

Comment:
@samlesu from [your comment](https://github.com/SubstraFoundation/substra-tests/pull/1/files#r339546891):

> I think the tests setup should stay independent from the backend setup

Following the same logic, would it make sense to hardcode the tmp path in the skaffold case, instead of pulling it from the skaffold file? 

i.e. 

```suggestion
        shared_path="/tmp/substra",
```

Reply:
All the network configuration is created from the skaffold file directly so I don't think it is a good idea to hardcode only the `shared_path`.

In my opinion, what could be done for docker-compose is to add a parser for a `docker-compose.yaml` file generating the test configuration.

Reply:
If I understand correctly, there would be 3 ways to launch the tests:

- with a `docker-compose.yaml` file
- with a `skaffold.yaml` file
- with a custom `config.yaml` file

Is that correct?

Reply:
My previous comment may not be very clear. Let me add a bit of context :)

Currently there are two different ways to specify the network configuration:
- create a values file and define explicitly all the values for each node (works for both docker-compose and skaffold)
- parse the backend skaffold file to generate the network configuration

From my point of view, we need to keep this two modes and the first mode should be independent from how the network is started.

Reply:
> If I understand correctly, there would be 3 ways to launch the tests:
> 
> * with a `docker-compose.yaml` file
> * with a `skaffold.yaml` file
> * with a custom `config.yaml` file
> 
> Is that correct?

Exactly.